testing/tests/network/manual_testing.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2019-2021 Aleo Systems Inc.
+// This file is part of the snarkOS library.
+
+// The snarkOS library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The snarkOS library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the snarkOS library. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::common::start_logger;
+
+use pea2pea::{
+    protocols::{Handshake, Reading, Writing},
+    Config,
+    Node as Pea2PeaNode,
+};
+use snarkos_testing::TestNode;
+
+use std::net::{IpAddr, Ipv4Addr};
+
+/// This test is intended to be run manually in order to monitor the behavior of
+/// a full snarkOS node started independently.
+#[ignore]
+#[tokio::test]
+async fn spawn_inert_node_at_port() {
+    start_logger();
+
+    // FIXME: snarkOS currently expects every node to be on this port, making it difficult
+    // to run local tests with multiple nodes.
+    const PORT: u16 = 4135;
```

Comment:
Does snarkOS currently assign a port if it isn't specified?

Reply:
`snarkOS` currently uses and expects `4132` (for testnet2); this is just a placeholder that I used for manual tests (which required minor tweaks to the node) until these rules are relaxed (at least for test purposes)